templates/registration/login.html

```diff
@@ -1,12 +1,17 @@
+<!-- templates/registration/login.html -->
 {% extends 'base.html' %}
+
 {% block title %}Login{% endblock %}
+
 {% block content %}
 <h2 class="text-center">Login</h2>
 <div style="text-align:center">
   <form method="post" id="form_login">
     {% csrf_token %}
     {{ form.as_p }}
     <button class="btn btn-dark" type="submit">Login</button>
+
+    <p><a href="{% url 'password_reset' %}">reset password</a></p>
   </form>
 </div>
 {% endblock %}
```

Comment:
A comment about a logout button: it sprawls outside of a window, can you make it inside a window?